qiita_pet/templates/admin_processing_job.html

```diff
@@ -2,6 +2,116 @@
 
 {% block head %}
 <script type="text/javascript">
+  var jobsTable;
+
+  // This modal view is used to display job details when a job has errored or
+  // succeeded. Needs to be in the main scope because this function is called
+  // from the "onclick" attribute of some elements.
+  var populateModal = function(title, main) {
+    $('#job-output-modal').find('[name="modal-title"]').html(title);
+    $('#job-output-modal').find('[name="modal-main-text"]').html(unescape(decodeURIComponent(main)));
+  }
+
+  // Initialize the previous jobs table once the document is ready
+  $(function() {
+    var renderJobArguments = function(data, type, row, meta) {
+      // select how you want to render the data
+      var lines = [], container;
+
+      for (key in data) {
+        lines.push('<b>' + key + '</b>:');
+        if (typeof data[key] === 'string'){
+          lines.push(data[key]);
+        }
+        else if (typeof data[key] === 'object') {
+          container = data[key];
+
+          // if we have all these attributes then render a named hyperlink to
+          //download the data
+          if (container['body'] !== undefined &&
+              container['filename'] !== undefined &&
+              container['content_type'] !== undefined) {
+              lines.push('<a download="' + container['filename'] +
+                         '" href="data:text/plain;charset=utf-8,' +
+                         encodeURIComponent(container['body']) + '">' +
+                         container['filename'] + '</a>');
+          }
+          else {
+            lines.push('JavaScript Object')
```

Comment:
Is this right?

Reply:
Yes, for objects that aren't strings or files, I figure we shouldn't even try. I updated this string to read "Unsupported JavaScript Object".